content/en/blog/2025/autonomus_testing_with_antithesis.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Autonomous Testing of etcd's Robustness
+author: "Marek Siarkowicz, Google & Craig Alfieri, Antithesis"
+date: 2025-08-13
+draft: false
+---
+
+Ensuring the reliability of etcd is our top priority. As a critical component of many production systems, we are continuously improving our testing methodologies. This blog post describes how we used autonomous testing to uncover subtle bugs, validate the robustness of our releases, and increase our confidence in etcd's stability. We'll share our key findings and how they have improved etcd.
+
+## Enhancing etcd's Robustness Testing
+
+Etcd is a critical component in many production systems, most notably as the primary datastore for Kubernetes. After some issues with the v3.5 release, the etcd maintainers developed a new [robustness testing framework](https://github.com/etcd-io/etcd/issues/14045) to better test for correctness under various failure scenarios. To further enhance our testing capabilities, we integrated an autonomous testing platform from [Antithesis](https://antithesis.com/) into our workflow. This allowed us to perform deterministic simulation testing for our distributed system.
+
+Our goals for this testing effort were to:
+
+1.  **Validate the robustness of etcd v3.6.**
+2.  **Improve etcd's software quality by finding and fixing bugs.**
+3.  **Enhance our existing testing framework with autonomous testing.**
```

Comment:
The linter is complaining about the spaces after the bullet points or the numbers.

```suggestion
1. **Validate the robustness of etcd v3.6.**
2. **Improve etcd's software quality by finding and fixing bugs.**
3. **Enhance our existing testing framework with autonomous testing.**
```